wordpress-mobile/gutenberg-mobile
# Correct color palette in color settings

Fixes #2222 

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22138

To test:

1. Open mobile app
2. Add Buttons block
3. Expect to see not duplicated colors 

<img width="300" src="https://user-images.githubusercontent.com/22746080/81183728-cb3c5900-8faf-11ea-9aac-af5baea20a3a.gif" />


PR submission checklist:
